### PR TITLE
Binding Problem Solved

### PR DESCRIPTION
**Pull request by marianbme**

### CODE REVIEW

File: StreetCare/StreetCare/Views/Screens/OutreachFormView.swift [hunk 1]
```diff
@@ -4,10 +4,10 @@
 //
 //  Created by Marian John on 11/12/24.
 //
-import SwiftUI
-import FirebaseFirestore
+//import SwiftUI
+//import FirebaseFirestore
 
-struct OutreachFormView: View {
+/*struct OutreachFormView: View {
```
Comment (phillipsk) [inline StreetCare/StreetCare/Views/Screens/OutreachFormView.swift:10]: Please remove the old code, don't leave it as a comment

File: StreetCare/StreetCare/Views/Screens/OutreachFormView.swift [hunk 2]
```diff
@@ -254,7 +254,195 @@ struct OutreachFormView: View {
...
+                    NavLinkButton(title: NSLocalizedString("discardButtonTitle", comment: ""), width: 300, secondaryButton: true)
+                        .onTapGesture {
+                            isPresented = false // Close both views on discard
+                        }
+                }
+                .frame(maxWidth: .infinity, alignment: .center)
+                .padding()
+            }
+            .padding()
+        }
```
Comment (phillipsk) [inline StreetCare/StreetCare/Views/Screens/OutreachFormView.swift:424]: Is the code for the select skills sheet and overlay missing? Please review the previous working code and ensure it's still carried over into your new changes

File: .DS_Store
Comment (phillipsk) [inline .DS_Store:1]: Remove this file

**Review by phillipsk — CHANGES_REQUESTED** [review]
Please review the previous working code and ensure it's still carried over into your new changes

**Comment by marianbme** [comment]
Solved the issue, deleledted .DS store file and removed old code

File: StreetCare/StreetCare/Views/Screens/OutreachFormView.swift [hunk 3]
```diff
@@ -308,7 +57,7 @@ struct OutreachFormView: View {
             isLoading = false
             alertMessage = "Event saved successfully!"
```
Comment (phillipsk) [inline StreetCare/StreetCare/Views/Screens/OutreachFormView.swift:58]: Does this alert message display after saving an event?

**Review by phillipsk — CHANGES_REQUESTED** [review]

**Comment by phillipsk** [comment]
Merge conflicts need to be resolved

**Review by phillipsk — CHANGES_REQUESTED** [review]
After a user successfully creates an outreach event an alert displays indicating the submission was successful
1) Update the alert message title to "New Outreach Event" remove generic `Message` placeholder text
image


2) After the user taps OK the previous screen should be dismissed and user should return to the community view of help requests

**Comment by phillipsk** [comment]
> `StreetCare/Pods/Manifest.lock`
> `StreetCare/Pods/Pods.xcodeproj/project.pbxproj`
> `StreetCare/Pods/Target Support Files/Pods-StreetCare/Pods-StreetCare.debug.xcconfig`
> `StreetCare/Pods/Target Support Files/Pods-StreetCare/Pods-StreetCare.release.xcconfig`

Try and resolve these merge conflicts

**Review by phillipsk — APPROVED** [review]

**Pull request merged**